Fantasy Scouts Web App/login.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<!--[if lt IE 9]>
+<script src="http://html5shim.googlecode.com/svn/trunk/html5.js">
+</script>
+<![endif]-->
+<title>Forms</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, inital scale =1.0">
+<link rel="stylesheet" href="ganggreen.css">
+</head>
+<body>
+<div id="container">
+<header>
+<h1>Fantasy Scouts</h1>
+</header>
+<nav id="noNavPrint">
+<a href="index.html">Home</a> &nbsp;
+<a href="analyst.html">Analyst Hub</a> &nbsp;
+<a href="chat.html">Chat</a> &nbsp;
+<a href="search.html">Search</a>
```

Comment:
Omit site links on login, add in continue as guest option to navigate to site index